steps/oci-compute-step-instances-describe/step.yaml

```diff
@@ -0,0 +1,61 @@
+# The schema version. Required. Must be exactly the string "integration/v1".
+apiVersion: integration/v1
+
+# The schema kind. Required. Must be one of "Query", "Step", or "Trigger"
+# corresponding to its directory location.
+kind: Step
+
+# The name of the action. Required. Must be exactly the name of the directory
+# containing the action.
+name: oci-compute-step-instances-describe
+
+# The version of the action. Required. Must be an integer. If specified in the
+# directory name, must be exactly the version in the directory name.
+version: 1
+
+# High-level phrase describing what this action does. Required.
+summary: Describe OCI Compute instances
+
+# Single-paragraph explanation of what this action does in more detail.
+# Optional. Markdown.
+description: |
+  Lists the instances in the provided OCI region.
+
+# URL or path relative to this file to an icon or icons representing this
+# action. Optional. Defaults to the integration icon.
+icon:
+
+# The mechanism to use to construct this step. Required. Must be an action
+# builder. See the Builders section below.
+build:
+  # The schema version for builders. Required. For now, must be the exact
+  # string "build/v1". We may consider supporting custom third-party builders
+  # in the future.
+  apiVersion: build/v1
+
+  # The builder to use. Required.
+  kind: Docker
+
+publish:
+  repository: relaysh/oci-compute-step-instances-describe
+
+schemas:
+  spec: 
+    source: file
+    file: spec.schema.json
+  output:
+    source: file
+    file: output.schema.json
+
+examples:
+- summary: Describe all OCI Compute instances
+  content:
+    apiVersion: v1
+    kind: Step
+    name: describe-instances
+    image: relaysh/oci-compute-step-instances-describe
+    spec:
+      oci:
+        connection: !Connection { type: oci, name: my-oci-account }
```

Comment:
We will have to add the OCI connection type on our side. I will open a ticket for it.